src/common/gpx.c

```diff
@@ -96,7 +96,7 @@ dt_gpx_t *dt_gpx_new(const gchar *filename)
   gpx = g_malloc0(sizeof(dt_gpx_t));
 
   /* skip UTF-8 BOM */
-  if(gpxmf_size > 3 && gpxmf_content[0] == '\xef' && gpxmf_content[1] == '\xbb' && gpxmf_content[2] == '\xbf')
```

Comment:
I don't understand this. It may be the case that the file is less than 3 chars and so the following checks will overrun the buffer to me.

Reply:
Cannot occur because of line 93:
`  if(!gpxmf_content || gpxmf_size < 10) goto error;`

Reply:
Missed that, sorry !